media-gfx/freewrl/files/freewrl-3.0.0-fno-common.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/lib/desktop.c
++++ b/src/lib/desktop.c
+@@ -233,7 +233,7 @@ enum {
+ 	url2file_task_spawn,
+ } url2file_task_tactic;
+ 
+-enum {
++extern enum {
```

Comment:
try turning this into a `typedef` instead

Reply:
Done